### DISCUSSION TITLE
How does this differ from other things?

### DISCUSSION THREAD

**Discussion by gvanrossum** [python/core-sprint#19] [General]
Is it more like issues? How does it differ?

Can we use it as an alternative to Discourse?

**Comment by zware** [comment]
It seems to use the same identifier sequence as issues (and pull requests), but without labels or other tracking metadata.  The only thing I can see that's *added* compared to issues is the "answered/unanswered" marker.  Is that only settable by the person who opened the discussion (or presumably admins)?  I for one don't appear able to change it on this one.

**Reply by zware** [reply]
Ah, it looks like a comment can be marked as being the answer to the discussion, which I am actually able to do with my comment (presumably because I'm a collaborator?).  This is looking rather more StackOverflow-y than Discourse-y, but it might be usable in place of Discourse anyway.  I suspect it may be even worse about marking how far you've read, though.

**Comment by gvanrossum** [comment]
I use email for that. :-)

**Comment by zware** [comment]
This does seem like it may be more friendly to email interaction than
Discourse :).  I use Discourse's "mailing list mode", but it has a
significant delay, sometimes decides to include *way* too much context, and
doesn't do particularly well with replies via email.

I am a bit curious where exactly this reply via email will wind up :)

**Comment by freddrake** [comment]
Another not-quite-email thing won't make anything better.

**Comment by Mariatta** [comment]
We can convert an issue to a discussion.

The way I see it:
An issue get resolved by pull request.
A discussion item get resolved by an answer.

**Reply by freddrake** [reply]
So this is really a way to push support requests out of the bucket of bug reports and feature requests.

I can certainly see value in that, but... still leaves me thinking this is a half-baked StackOverflow replacement.  Possibly useful, but calling it Discussions is misleading.  Interesting discussions rarely lead to a simple "yes, this has been answered."

(Yes, I realize the Python core team didn't name "Discussions"; not suggesting this particular failure is on us.)

But the mis-nomer provides a difficult-to-avoid problem in understanding for newcomers, for something where being welcoming is a real goal.

**Reply by pitrou** [reply]
It certainly does look like a StackOverflow replacement (half-baked or not, it depends on what you count as the essential features of StackOverflow).
